pkg/user/apiserver/registry/useridentitymapping/rest.go

```diff
@@ -320,9 +319,10 @@ func unsetIdentityUser(identity *userapi.Identity) bool {
 func mappingFor(user *userapi.User, identity *userapi.Identity) (*userinternal.UserIdentityMapping, error) {
 	return &userinternal.UserIdentityMapping{
 		ObjectMeta: metav1.ObjectMeta{
-			Name:            identity.Name,
-			ResourceVersion: identity.ResourceVersion,
-			UID:             identity.UID,
+			Name:              identity.Name,
+			ResourceVersion:   identity.ResourceVersion,
+			UID:               identity.UID,
+			CreationTimestamp: metav1.Now(),
```

Comment:
the mapping is defined by the existence of the identity so I _think_ that the more correct mapping creation is defined by `identity.CreationTimestamp`

Reply:
ok, will update to the identity one